### DISCUSSION TITLE
Hello大家好，我是项目作者。

### DISCUSSION THREAD

**Discussion by mengchaoheng** [mengchaoheng/SCUT_thesis#9] [General]
刚刚看了一下2022的论文撰写规范，好家伙就只改了年份。最近使用了Mac进行编译，字体部分更新了一下。不知道大家有什么使用上的困难，不管是latex也好，git也好，都可以进来讨论。如果有需要我新增一份简短的git说明。

**Comment by mengchaoheng** [comment]
有问题可以在[讨论区](https://github.com/mengchaoheng/SCUT_thesis/discussions)new discussion新增一个讨论，这样大家都能看到。发邮件私信的话我不得不每次都回答一样的问题，效率不高。而且在GitHub上的一切动态都是会收到邮件的，推送很及时不会延迟。

**Comment by Komari-Koshigaya** [comment]
> 刚刚看了一下2022的论文撰写规范，好家伙就只改了年份。最近使用了Mac进行编译，字体部分更新了一下。不知道大家有什么使用上的困难，不管是latex也好，git也好，都可以进来讨论。如果有需要我新增一份简短的git说明。

实际上“攻读学位期间取得的研究成果”模板 也改了几个字
2020版本 ![image](https://user-images.githubusercontent.com/39209375/163087107-d87c49a7-624e-489f-9a17-f94888e37b51.png)

2022版本 
![image](https://user-images.githubusercontent.com/39209375/163087272-9f6c1ec4-04a0-4722-9c5b-09c0be4d58b1.png)

忘记跟你说了  好家伙，我自己也没改😅

**Reply by mengchaoheng** [reply]
好家伙，学校这是玩我们呢哈哈哈。我都不知道怎么通知用此模版的所有人（哭）

**Comment by mengchaoheng** [comment]
我上传了最新提交，我建议按照讨论区的vscode环境配置安装vscode，方便查看最新提交进行了哪些更更改。抱歉我左思右想还是不要进行大改，容易加入更多的bug。仅仅把字号设置正确就好了。所以.cls文件我改了：
```Console
% 很多字体字号并没有使用。第一个参数指定字号，第二个参数为baselineskip默认=1.2倍字号(pt). 结合\linespread可以实现控制行距。
% 但这里的设置暂时不清楚为什么，也暂时找不到其他方法。
% ToDo：使用更好的字体字号行距控制方法。例如用\usepackage{setspace}，然后\onehalfspacing实现单倍行距。但具体细节待考证。
...
\newcommand{\wuhao}{\fontsize{10.5pt}{13pt}\selectfont} % change 10.95pt to 10.5pt --by mch, 真正的五号字应该是10.5。10.95是通常所说的11pt，在英文排版中使用。

...
\newcommand{\pubfont}{\songti\wuhao\normalfont}          % 《攻读博士/硕士学位期间取得的研究成果》字体，2022年更新。
...
\linespacing{1.682}% 主行距～1.5倍  % 手调参数，利用1比1的打印效果看行距是否和word的1.5倍行距一样。latex行距控制水太深了。见109行ToDo。--by mch
```
没有进行实质性的更改主要考虑稳定压倒一切，在没有彻底弄清楚这些关系之前不敢乱动。

**Reply by mengchaoheng** [reply]
e78ca6f2999c0dc3623485a0e0c617413449032f
